src/constants.ts

```diff
@@ -239,3 +241,321 @@ export type STU3Resource =
     | 'TestReport'
     | 'ValueSet'
     | 'VisionPrescription';
+
+export const SUPPORTED_R4_RESOURCES: R4Resource[] = [

```

Comment:
I would change the names now to be 'BASE' not 'SUPPORTED' as the interface shouldn't be saying what the server supports or not

Reply:
fixed